app/src/main/java/com/practicum/playlistmaker/App.kt

```diff
@@ -0,0 +1,40 @@
+package com.practicum.playlistmaker
+
+import android.app.Application
+import androidx.appcompat.app.AppCompatDelegate
+import androidx.appcompat.app.AppCompatDelegate.MODE_NIGHT_NO
+import androidx.appcompat.app.AppCompatDelegate.MODE_NIGHT_YES
+
+
+private const val DARK_MODE = "dark_mode"
+const val PLAYLIST_MAKER_PREFERENCES = "PLAYLIST_MAKER_PREFERENCES"
+
+class App : Application() {
+    var darkTheme = false
+    private val sharedPreferences by lazy {
```

Comment:
```suggestion
    private val sharedPreferences by lazy(LazyThreadSafetyMode.None) {
```
Лучше сделать так, т.к. есть гарантии, что `App` объект инициализируется только 1 раз на одном потоке, поэтому здесь не нужна синхронизация для `lazy`, из-за синхронизации код дольше работает, поэтому лишнюю лучше убирать